package.json

```diff
@@ -4,7 +4,6 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "pretest": "gulp lint",
     "test": "nyc ava"
```

Comment:
e.g. `npm test` should be a prepush hook. but if you just want to run the tests simply use `ava`